core/backend/src/ElementTreeWalker.ts

```diff
@@ -0,0 +1,432 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Elements
+ */
+import { assert, DbResult, Id64Array, Id64String, Logger, LogLevel } from "@itwin/core-bentley";
```

Comment:
Need to export everything from ElementTreeWalker.ts from the core-backend.ts barrel.

Reply:
done